frontend/src/pages/Mesh/target/TargetPanelCommon.tsx

```diff
@@ -164,6 +166,7 @@ export const renderNodeHeader = (
           {data.infraName}
 
           {renderHealthStatus(data)}
+          {serverConfig.ambientEnabled && data.infraName === 'istiod' && <AmbientLabel tooltip={tooltipMsgType.mesh} />}
```

Comment:
```suggestion
          {serverConfig.ambientEnabled && data.infraType === MeshInfraType.ISTIOD && <AmbientLabel tooltip={tooltipMsgType.mesh} />}
```

The name can change.